content/en/security/vulnerability_pipeline/mute.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Mute Rules
+---
+
+{{< callout url="https://www.datadoghq.com/product-preview/security-automation-pipelines/" >}}
+  Vulnerability Pipeline is in Preview. To enroll in the Preview for Mute rules, click <strong>Request Access</strong>.
+{{< /callout >}} 
+
+Configure mute rules to streamline security alerts by automatically filtering out non-urgent findings. This approach helps reduce noise from known false positives and accepted risks, allowing you to focus on addressing the most critical threats.
+
+## Create a mute rule
+
+1. On the [Vulnerability Pipeline][2] page, click **Add a New Rule** and select **Mute**.
+1. Enter a descriptive name for the rule, for example, Cloud Infrastructure Anomaly Warnings.
+1. Use the following boxes to configure the rule criteria:
+    - **Any of these types**: The types of findings that the rule should check for. Available types include **Misconfiguration**, **Attack Path**, **Identity Risk**, and **API Security Finding**.
+    - **Any of these tags or attributes**: The resource tags or attributes that must match for the rule to apply.
```

Comment:
Let me think on this. Must is more appropriate than "can", because these are required attributes that must be present on a resource in order for the rule to trigger. But I don't think I'm capturing the nuance as far as what happens when there are multiple values present for this field (where it becomes an "any" of the following values "must" be present).